### PR TITLE
Update Roadmap to mention Vega is completed

### PR DESCRIPTION
**Pull request by nooomski**
And some little things to make sure the page is parsed well for the new cosmos hub roadmap page.

### CODE REVIEW

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 1]
```diff
@@ -65,20 +66,21 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
   - Allows independent modules like Gravity DEX or Bridge to live on separate chains with their own development cycles.
 - Chain Name Service
-  -  Chain-ID registry
-    - Node registry
-    - IBC Path Resolution
-  -  Asset registry
-  -  Account registry
-  -  Bech32 registry
+  - Chain-ID registry
```
Comment (jackzampolin) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:69]: Nice!

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 2]
```diff
@@ -50,10 +55,6 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - Features include enabling transfer of rewards and voting rights
 - Items still being discussed:
   - Governance permissioned CosmWASM instance on the hub
-  - Cosmos <> Ethereum Gravity Bridge
-    - Transfer ATOM, ETH, ERC-20, and tokens on the Cosmos Hub between Ethereum- and Cosmos-compatible chains  
-    - Fee and reward model hosted across Cosmos and Ethereum
-    - Adds light-weight infrastructure and operational requirements with minimal slashing conditions to all Hub validators
   - Budget Module
     - Inflation funding directed to arbitrary module and account addresses
   - Farming Module
```
Comment (hxrts) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:60]: I think we can remove farming, but may want to double check

Comment (hxrts) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:60]: I think we can remove farming but may want to double check.

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 3]
```diff
@@ -65,20 +66,21 @@ The upgrades aim to add features such as liquidity, economic security, usability
...
-  -  Bech32 registry
+  - Chain-ID registry
+  - Node registry
+  - IBC Path Resolution
+  - Asset registry
+  - Account registry
+  - Bech32 registry
 
 
 ## Lambda Upgrade (expected Q3 2022)
```
Comment (hxrts) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:77]: IBC queries @jackzampolin ?

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 4]
```diff
@@ -13,19 +13,24 @@ The upgrades aim to add features such as liquidity, economic security, usability
...
- - IBC packet forward middleware v1.0.1
+  - The Gravity DEX with updates for dependencies
+- IBC v2.0.0
+- Tendermint v0.34.14
+- Cosmosvisor v0.1.0
+- IBC packet forward middleware v1.0.1
+  - Cosmos Hub as a router
+
+- External chain launch: Gravity Bridge
+  - Transfer ATOM, ETH, ERC-20, and tokens on the Cosmos Hub between Ethereum- and Cosmos-compatible chains
```
Comment (okwme) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md]:
```suggestion
  - Transfer ATOM, ETH, ERC-20, and other Cosmos tokens between Ethereum and the Gravity Bridge Chain and by extension all IBC connected chains.
```

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 2 above)
Comment (okwme) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:60]: would like to wait until there's possible public confirmation.

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 3 above)
Comment (okwme) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:77]: need to link up w IBC team to confirm their roadmap before speaking on their behalf. Or is the query work confirmed to come from an external group??

**Pull request merged**